### PR TITLE
Update troubleshooting document

### PR DESCRIPTION
**Pull request by n-riesco**

### CODE REVIEW

**Comment by lgeiger** [comment]
Awesome thanks for going through all the issues! 👍

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  print sys.executable
+  ```
+
+  Let's say this script prints out `c:\Anaconda2\python.exe`. You can configure
+  Atom to use this executable by running
+  `apm config set python c:\Anaconda2\python.exe`. Then, Hydrogen can be
+  installed by running `apm install hydrogen`.
+
+
+### No kernel for language X found, but I have a kernel for that language.
```
Comment (lgeiger) [inline TROUBLESHOOTING.md]: Should we add an item about the `Language Mappings` setting if the kernel uses a different name (e.g. `scala211`)?

Comment (n-riesco) [inline TROUBLESHOOTING.md]: I haven't tested #355 yet. Shall we wait until then?

Comment (lgeiger) [inline TROUBLESHOOTING.md]: Oh I forgot about #355.

**Pull request merged**